origin/src/lib.rs

```diff
@@ -34,6 +36,10 @@ pub use threads::{
     default_guard_size, default_stack_size, detach_thread, join_thread, thread_stack, Thread,
 };
 
+#[cfg(target_vendor = "mustang")]
+/// must be used as the global allocator on mustang
+pub use allocator::GlobalAllocator;
```

Comment:
If origin only works with this allocator, would it make sense to move the `#[global_allocator]` attribute into origin?

Reply:
it doesn't work, thats what i tried in 
31ae005 , and changed in 
0603b8b , the linking issue in question was caused by moving global allocator to mustang. looked like the symbols from `unwinding` where not included.